Gemfile.lock

```diff
@@ -61,60 +64,59 @@ GEM
       selenium-webdriver
     childprocess (4.1.0)
     coderay (1.1.3)
-    concurrent-ruby (1.1.8)
-    diff-lcs (1.4.4)
-    faraday (1.3.0)
-      faraday-net_http (~> 1.0)
-      multipart-post (>= 1.2, < 3)
-      ruby2_keywords
-    faraday-net_http (1.0.1)
-    hashie (4.1.0)
-    i18n (1.8.9)
+    concurrent-ruby (1.1.10)
+    diff-lcs (1.5.0)
+    faraday (2.6.0)
```

Comment:
faraday 2.0 changes how basic auth is used, and our code uses the old way:

https://lostisland.github.io/faraday/middleware/authentication#basic-authentication